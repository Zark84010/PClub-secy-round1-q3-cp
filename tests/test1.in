5 78 9
36 48 24 12 89
f 1 4
f 2 3
f 1 5
c 5 90
f 1 5
f 4 5
c 5 84
c 4 70
f 4 5
