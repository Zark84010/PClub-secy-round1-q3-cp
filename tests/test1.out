924
1848
78
78
78
78
